app/views/moderator.py

```diff
@@ -233,12 +233,20 @@ def post(self, request):
         return redirect('manage_oz')
 
 @method_decorator(moderator_required, name='dispatch')
-class AddSignupView(View):
+class ManageSignupView(View):
     def get(self, request):
         return redirect('manage_players')
 
     def post(self, request):
         game = most_recent_game()
+
+        locations = SignupLocation.objects.filter(game=game)
+        for location in locations:
+            if str(location.id)+'-remove' in request.POST:
+                messages.success(request, f"Deleted signup location {location}")
```

Comment:
Nit: You should probably delete and then tell them that it was deleted just in case something throws an exception in between but it shouldn't matter too much.